docs/scripting/language/Variables

```diff
@@ -0,0 +1,218 @@
+---
+id: variables
+title: "Variables"
+---
+
+# Variables
+
+  One of the most important concepts in programming is the concept of ‘variables’. In programming, a variable is an entity that is changeable, but in terms of what ? In Pawn language a variable holds a ‘value’ at any time and that value-as the name suggests-is ‘variable’ or ‘changeable’.
+
+  The reason why variables are so important is because they are basically small units of computer memory which can hold or ‘remember’ different values while the program is under execution (running), and that property turns out to be very useful in programming. For example, you want to keep track of the scores of 100 players in a game, you can do it easily by programming the computer to store (remember) and update those values. Later if you want to find the mean score of those players or want to create a leaderboard, those values from the variables can be easily accessed and used for that purpose.
+
+## Declaring Variables
+
+Following is the syntax for variable declaration :
+
+```c
+// Creating (more appropriately, 'declaring') a variable named 'my_variable'
+
+new my_variable ;
```

Comment:
Please follow the [Style Guide](https://www.open.mp/docs/scripting/language/Style).

Reply:
It is necessary to follow naming conventions for the wiki ?

Reply:
Yes, coinsistency is important for newcomers.

Reply:
Corrected them.